src/doc/man/cargo-generate-lockfile.md

```diff
@@ -32,6 +32,14 @@ lockfile and has more options for controlling update behavior.
 
 {{> options-ignore-rust-version }}
 
+{{#option "`--publish-time` _yyyy-mm-ddThh:mm:ssZ_" }}
+Latest publish time allowed for registry packages (Unstable)
+
+This is a best-effort filter on allowed packages, including:
+- packages from unsupported registries are always accepted
+- only the latest yank status is respected
```

Comment:
I had a hard time understanding this. Do need a test with yanked versions?

Reply:
`cargo generate-lockfile` will only look at the current yank status, independent of `--publish-time`.  In your scenario, it is currently yanked so that is the state that will be used.

Reply:
Made a small tweak but likely not covering what you are looking for.  Maybe it is curse of knowledge but I'm having a hard time thinking of how to further clarify this.  maybe provide an example?